<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HandSilent - Register</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">    QMainWindow {
        background-color: #353535;
    }
    QLabel {
        color: black;
        font-size: 16px;
    }
    QPushButton {
        background-color: #2a2a2a;
        border: 2px solid #3a3a3a;
        border-radius: 8px;
        color: white;
        font-size: 14px;
        padding: 8px;
        margin: 4px;
    }
    QPushButton:hover {
        background-color: #4a4a4a;
        border-color: #5a5a5a;
    }
    QPushButton:pressed {
        background-color: #1a1a1a;
        border-color: #3a3a3a;
    }
    QProgressBar {
        background-color: #2e2e2e;
        border-radius: 8px;
        color: white;
        font-size: 14px;
    }
    QProgressBar::chunk {
        background-color: #42a5f5;
        border-radius: 8px;
    }
    QLineEdit {
        background-color: #2e2e2e;
        border: 1px solid #3a3a3a;
        border-radius: 6px;
        color: gray;
        padding: 4px;
    }
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-top-left-radius: 20px;
border-top-right-radius: 20px;
border-bottom-left-radius: 20px;
border-bottom-right-radius: 20px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Register</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Bắt đầu hành trình giao tiếp không lời nói</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Đăng ký ngay hôm nay!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_new_email">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>Email or Username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_new_password">
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Password (6 or more characters)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_agree_and_join">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(160, 237, 83);</string>
     </property>
     <property name="text">
      <string>REGISTER</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
